docs/developer-documentation.md

```diff
@@ -171,7 +171,7 @@ After installing Poetry, run the following commands:
 ```bash
 # Install the dependencies
 $ poetry install --only=dev-host
-$ poetry run flake8 $(git ls-files '**/*.py')
+$ poetry run flake8 $(git ls-files '*.py')
```

Comment:
Don't  change this -- those two things are different.

Reply:
Yes, those two are different. In that case, `**/*.py` has less coverage than `*.py`. I tested it before and just now.

![image](https://github.com/user-attachments/assets/95eb7a6e-ec93-413c-ae29-dd269e75f6b9)

`*.py` is what's being used in GitHub actions at the moment:

https://github.com/Screenly/Anthias/blob/70afe9ba49d42f8a050cce2faaf2a963cd4a8bb7/.github/workflows/python-lint.yaml#L47